CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added a new function `merge_sync`. In this function a write will be flushed from the operating system buffer cache before the write is considered complete. (#368)
 - Added conversion into boxed values for values which implement `Service` or `Transaction` traits. (#366)
 - Added constructor for the `ServiceContext` which can be useful for the alternative node implementations. (#366)
+- Added serialization support floating point types (`f32` and `f64`). Note: special values (Infinity and NaN) aren't supported. (#384)
```

Comment:
I don't quite get what "unsupported" means here. Take the following code:

```rust
use std::f64;
use messages::Message;

message! {
    struct F64Message {
        const TYPE = 0;
        const ID = 0;
        const SIZE = 8;
        field foo: f64 [0 => 8]
    }
}

let sk = SecretKey::new([1; 64]);
let x = F64Message::new(f64::INFINITY, &sk);
let buffer = x.raw().clone();
let y = F64Message::from_raw(buffer).unwrap();
assert_eq!(x, y);
```

It fails on `unwrap()`, which means that it's now possible to create and serialize unserializable messages. Honestly doesn't look like a good design decision to me. Is non-support of infinities and NaNs temporary? Otherwise, maybe it could make sense to provide `Field` implementation for *wrappers* of `f32` / `f64`, which are guaranteed not to equal infinities or NaNs?